apps/web-giddh/src/app/new-inventory/component/stock-create-edit/stock-create-edit.component.html

```diff
@@ -431,6 +431,12 @@
                                                                                 class="icon-button-remove"
                                                                                 type="button"
                                                                                 (click)="deletePurchaseUnitPrice(i)"
+                                                                                *ngIf="
+                                                                                    i !==
+                                                                                    stockForm.purchaseAccountDetails
```

Comment:
apply condition to delete only if records are more than 1

Reply:
done